rocAL/rocAL/include/master_graph.h

```diff
@@ -88,6 +90,12 @@ class MasterGraph
     void create_randombboxcrop_reader(RandomBBoxCrop_MetaDataReaderType reader_type, RandomBBoxCrop_MetaDataType label_type, bool all_boxes_overlap, bool no_crop, FloatParam* aspect_ratio, bool has_shape, int crop_width, int crop_height, int num_attempts, FloatParam* scaling, int total_num_attempts, int64_t seed=0);
     const std::pair<ImageNameBatch,pMetaDataBatch>& meta_data();
     void set_loop(bool val) { _loop = val; }
+    void set_output_images(std::vector<Image*> output_images, unsigned int num_of_outputs)
```

Comment:
Please pass vector by reference or pointer to avoid copy. Also if it is read only, please pass const reference

Reply:
Done